### ISSUE TITLE
Progress Watchdog, hardcoded default Namespace

### ISSUE THREAD

**Issue by splattner** [iteratec/multi-juicer#47]
https://github.com/iteratec/multi-juicer/blob/master/progress-watchdog/main.go#L78 there is the "default" Namespace hardcoded which does not work if deployed to any other namespace than "default".
 
Therefore I also see permission errors (as sa, role, rolebinding in an other namespace):

```
panic: deployments.apps is forbidden: User "system:serviceaccount:juicy-ctf:progress-watchdog" cannot list resource "deployments" in API group "apps" in the namespace "default"
goroutine 1 [running]:
main.createProgressUpdateJobs(0xc000180420, 0xc000271760)
/src/main.go:80 +0x5da
main.main()
/src/main.go:66 +0x2ff
```

**Comment by J12934** [comment]
Oh, that's a good catch. Thank you 👍
Will try to push out a fix to the helm chart later today.

**Comment by J12934** [comment]
Fixed in [v3.1.1](https://github.com/iteratec/multi-juicer/releases/tag/v3.1.1) 🎉
Thanks again for reporting 👍

**Comment by splattner** [comment]
Awesome! Thx for the fast fix!
